iothub/device/src/Authentication/DeviceAuthenticationWithTokenRefresh.cs

```diff
@@ -11,54 +11,16 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public abstract class DeviceAuthenticationWithTokenRefresh : AuthenticationWithTokenRefresh
     {
-        internal const int DefaultTimeToLiveSeconds = 1 * 60 * 60;
-        internal const int DefaultBufferPercentage = 15;
-
         /// <summary>
-        /// Initializes a new instance of the the class using default TTL and TTL buffer time settings.
+        /// Initializes a new instance of the <c>DeviceAuthenticationWithTokenRefresh</c> class.
```

Comment:
We don't do this (<c>className</c>) anywhere else. Can we just resort to "Creates a new instance of this class."? It is generic, accurate, and makes sense in the context of the object you are looking at, whether in the IDE or in web SDK docs.

Reply:
Thanks for pointing it out. I've tried cleaning this up as much as I could but there could be other instances left behind. We'll have to keep cleaning them up as we go.